### PR TITLE
Fetch

### PR DESCRIPTION
**Pull request by tRomanowski**
Hi there! 
This is what the app is doing!
The app has a "Randomize"-button.  With a click on the button a recipe with random ingredients is fetched from the API and it returns the id of the recipe. With the id returned a new fetch is started and this fetch returns the detailed recipe. This is now displayed as a card in a list.  If I click the button again the list gets a second card with the new recipe and so on!

### CODE REVIEW

File: src/components/RecipeList.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Recipe from './Recipe';
+import styled from 'styled-components';
+
+export default function RecipeList({ recipes }) {
+  return (
+    <StyledList>
```
Comment (Sven-Lyco) [inline src/components/RecipeList.js:7]: You should a role to your list :) 
```js
<StyledList role="list">
```

File: src/components/RecipeList.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          <li key={recipe.id}>
+            <Recipe
+              title={recipe.title}
+              image={recipe.image}
+              text={recipe.summary}
+              url={recipe.sourceUrl}
+            />
+          </li>
+        );
+      })}
```
Comment (Sven-Lyco) [inline src/components/RecipeList.js:19]: Maybe you can try to use the map method without a return statement, if you want :)

**Review by Sven-Lyco — COMMENTED** [review]
Looks good to me, you did a good job here 🔥

File: src/App.js [hunk 3]
```diff
@@ -1,58 +1,41 @@
...
+    async function getNewRecipe() {
       try {
-        const recipesFromServer = await fetchRecipes();
-        console.log(recipesFromServer);
-        setRecipes(recipesFromServer);
-        setRandomNumber(Math.floor(Math.random() * 9));
+        const response = await fetch(urlIngredients);
+        const data = await response.json();
+        const recipeID = data[0].id;
+        console.log(recipeID);
```
Comment (julialoeschel) [inline src/App.js:21]: always delete unnecessary code ;)

**Review by julialoeschel — COMMENTED** [review]
good job!!!

**Pull request merged**